packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -149,6 +156,14 @@ class _SegmentState<T> extends State<_Segment<T>> with TickerProviderStateMixin<
 
   @override
   Widget build(BuildContext context) {
+    Alignment scaleAlignment;
```

Comment:
`final Alignment scaleAlignment;`

Reply:
There's a fourth state where both `isLeft` and `isRight` are true that seems unhandled?

Reply:
Sorry not sure I understand this correctly, how to be both left and right segment? For the segment that `isLeft`, it is the leftmost segment in the segmented control, and for `isRight`, it is the rightmost segment (assuming text direction is ltr).